src/modules/Dashboard/features/Participants/Participants.tsx

```diff
@@ -0,0 +1,564 @@
+      const { respondentOrSubjectId } = context || {};
+      if (!respondentOrSubjectId) return;
+
+      setRespondentKey(respondentOrSubjectId);
+      handleSetDataForAppletPage({ respondentOrSubjectId, key: FilteredAppletsKey.Editable });
+      setEditRespondentPopupVisible(true);
+    },
+    sendInvitation: ({ context }: MenuActionProps<RespondentActionProps>) => {
+      const { respondentOrSubjectId, email = null } = context || {};
+      if (!respondentOrSubjectId) return;
+
+      handleInviteClick({ respondentOrSubjectId, email });
+    },
+  };
+
+  const { execute: updateRespondentsPin } = useAsync(
+    updateRespondentsPinApi,
+    handleReload,
+    undefined,
+    () => setIsLoading(false),
+  );
+
+  const { execute: updateSubjectsPin } = useAsync(
+    updateSubjectsPinApi,
+    handleReload,
+    undefined,
+    () => setIsLoading(false),
+  );
+
+  const handlePinClick = ({ respondentId, subjectId }: HandlePinClick) => {
+    setIsLoading(true);
+    if (respondentId) {
+      updateRespondentsPin({ ownerId, userId: respondentId });
+
+      return;
+    }
+
+    updateSubjectsPin({ ownerId, userId: subjectId });
+  };
+
+  const editRespondentOnClose = (shouldReFetch: boolean) => {
+    setEditRespondentPopupVisible(false);
+    setChosenAppletData(null);
+    shouldReFetch && handleReload();
+  };
+
+  const removeRespondentAccessOnClose = (shouldReFetch?: boolean) => {
+    setRemoveAccessPopupVisible(false);
+    setChosenAppletData(null);
+    shouldReFetch && handleReload();
+  };
+
+  const handleInvitationPopupClose = (shouldReFetch?: boolean) => {
+    setInvitationPopupVisible(false);
+    setRespondentEmail(null);
+    shouldReFetch && handleReload();
+  };
+
+  const formatRow = (user: Respondent): Row => {
+    const {
+      secretIds,
+      nicknames,
+      lastSeen,
+      id: respondentId,
+      details,
+      isPinned,
+      isAnonymousRespondent,
+      status,
+      email,
+    } = user;
+    const latestActive = lastSeen ? timeAgo.format(getDateInUserTimezone(lastSeen)) : '--';
+    const schedule =
+      appletId && details?.[0]?.hasIndividualSchedule ? t('individual') : t('default');
+    const stringNicknames = joinWihComma(nicknames, true);
+    const stringSecretIds = joinWihComma(secretIds, true);
+    const respondentOrSubjectId = respondentId ?? details[0].subjectId;
+
+    return {
+      checkbox: {
+        content: () => (
+          <MuiCheckbox
+            checked={false}
+            icon={
+              <StyledCheckBox>
+                <Svg id="checkbox-empty-outline" height="20" width="20" />
+              </StyledCheckBox>
+            }
+            data-testid="dashboard-participants-checkbox"
+          />
+        ),
+        value: '',
+        width: ParticipantsColumnsWidth.Pin,
+      },
+      pin: {
+        content: () => <Pin isPinned={isPinned} data-testid="dashboard-respondents-pin" />,
```

Comment:
Great catch Paul! I hesitate to make this change as part of this ticket since the `Pin` component is used in a few places, and `outline_variant` is used in even more places. I'd rather make that change as part of a ticket where I can investigate the usages

What do you think about overriding the unpinned style for this specific usage?

Reply:
Hmm… I think it's probably best to leave this to a separate ticket rather than fragment this use-case, which I think will introduce unneeded complexity.

Reply:
Ok no prob, I'll create the Jira ticket. I ran into a hiccup and asked about it in Slack but I'll follow up with its creation once we iron that out